kits/flutter_ume_kit_dio/lib/src/pluggable.dart

```diff
@@ -5,15 +5,21 @@
 import 'dart:convert';
 
 import 'package:flutter/material.dart';
+import 'package:dio/dio.dart' show Dio;
 import 'package:flutter_ume/core/pluggable.dart';
 
+import 'models/http_interceptor.dart';
 import 'widgets/icon.dart' as icon;
 import 'widgets/pluggable_state.dart';
 
 // TODO(Alex): Implement [PluggableStream] for dot features.
 /// Implement a [Pluggable] to integrate with UME.
 class DioInspector extends StatefulWidget implements Pluggable {
-  const DioInspector({Key? key}) : super(key: key);
+  DioInspector({Key? key, required this.dio}) : super(key: key) {
+    dio.interceptors.add(UMEDioInterceptor());
+  }
```

Comment:
We guess a possibility, whether the user will have multiple dio instances. So here is a `List<Dio>`?

Reply:
I would prefer another constructor, such as `DioInspector.multiple(List<Dio> instances)`. I'll add this to the TODO list, since it requires more precise handle with multiple instances, e.g. tags, tabs, etc.